migration/src/m20230726_013107_remove_not_null_constraint_from_group_value.rs

```diff
@@ -0,0 +1,35 @@
+use digital_asset_types::dao::asset_grouping;
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .alter_table(
+                sea_query::Table::alter()
+                    .table(asset_grouping::Entity)
+                    .modify_column(ColumnDef::new(asset_grouping::Column::GroupValue).null())
+                    .to_owned(),
+            )
+            .await?;
+        Ok(())
+    }
+
```

Comment:
under what conditions does the group value become null?

Reply:
At one point we were setting group value to null to represent an unverified collection, but around the same time we made that change @danenbm added the verified/unverified.

However, it's worth keeping because it's the only way you can track collection removal for regular NFTs (and keep the slot_updated value). I don't think we handle collection removal right now, and it's quite rare in general.